test/apiv2/10-images.at

```diff
@@ -197,6 +197,11 @@ t POST "build?dockerfile=containerfile" $CONTAINERFILE_TAR 200 \
 t POST libpod/images/prune 200
 t POST libpod/images/prune 200 length=0 []
 
+# compat api must allow loading tar which contain multiple images
+podman pull docker.io/library/alpine:latest docker.io/library/busybox:latest
```

Comment:
This is going to break, we can't use `docker.io` in tests due to rate limiting.

@edsantiago Are there cached images in the apiv2 tests we can use? Otherwise, pulling our quay.io test images (a few are listed in https://github.com/containers/podman/blob/main/test/python/docker/compat/constant.py) should work

Reply:
`quay.io/libpod/busybox:latest`

Reply:
@mheon I don't understand the question. All the apiv2 tests use `quay.io/libpod/alpine`, `/busybox`, `/registry`. I see no reason why those couldn't be used here, too. Does that address your concern?

Reply:
Switched to quay.io

Reply:
@edsantiago Was just wondering if there was anything we could guarantee was already pulled so the test wouldn't need to pull two additional images

Reply:
Oh - I misunderstood "cached". No, right now the apiv2 tests don't do any caching optimization. They were intended as a very simple framework with which to test the API; network-pull stuff was supposed to be tested elsewhere. Looks like I need to rethink that.